app/gui2/src/components/GraphEditor.vue

```diff
@@ -202,8 +204,27 @@ const graphBindingsHandler = graphBindings.handler({
     if (keyboardBusy()) return false
     readNodeFromClipboard()
   },
+  enterNode() {
+    if (keyboardBusy()) return false
+    const selectedNode = set.first(nodeSelection.selected)
+    if (selectedNode) {
+      enterNode(selectedNode)
+    }
+  },
+  exitNode() {
+    if (keyboardBusy()) return false
+    exitNode()
+  },
 })
 
+const handleClick = useDoubleClick(
+  (e) => {
+    graphBindingsHandler(e)
+  },
+  () => {
+    exitNode()
+  },
+).handleClick
```

Comment:
I feel like we should add double-click support to the bindings system. Then it would become another aliased binding for the `exitNode` handler. But it doesn't necessarily need to be addressed in this PR, as its kinda unrelated.